packages/connect/src/base/entry.ts

```diff
@@ -59,10 +61,65 @@ type ConnResult = ProtocolStream & {
   conn: Connection
 }
 
-function latencyCompare(a: ConnectionResult, b: ConnectionResult) {
-  return a.entry.latency - b.entry.latency
+type Grouped = [id: string, results: (ConnectionResult | Error | undefined)[]]
+
+/**
+ * Sort results ascending in latency
```

Comment:
This is a comparison function, not a list sorting function.

Reply:
thx, outdated comments, changed